src/main/java/net/dv8tion/jda/api/entities/PermissionOverride.java

```diff
@@ -100,6 +101,63 @@
     @Nonnull
     JDA getJDA();
 
+    /**
+     * This method will return the {@link net.dv8tion.jda.api.entities.IPermissionHolder PermissionHolder} of this {@link net.dv8tion.jda.api.entities.PermissionOverride PermissionOverride}.
```

Comment:
You shouldn't link to the same class in javadoc, its redundant.

Reply:
Same as above.